bin/travis_setup.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+# Strict mode
+set -euo pipefail
+ORIG_IFS="$IFS"
+IFS=$'\n\t'
```

Comment:
The build should fail if one of the commands in this script fails. This is not currently the case and `set -e` fixes that. As for the other flags I invite you to read [this](http://redsymbol.net/articles/unofficial-bash-strict-mode/), the gist is that it hardens the script.